packages/world/src/VoxelCoord.sol

```diff
@@ -315,13 +317,12 @@ library VoxelCoordLib {
       Position._set(entityId, coord.x, coord.y, coord.z);
       ReversePosition._set(coord.x, coord.y, coord.z, entityId);
       ObjectType._set(entityId, objectTypeId);
-
-      Mass._setMass(entityId, ObjectTypeMetadata._getMass(objectTypeId));
```

Comment:
I moved this mass check outside since this function is used for non-base entities as well. Thats why also made it return a bool indicating its this code path

Reply:
I feel we should rethink this function, it is hard to reason about what it is doing and what's the bool value for

Reply:
hmm the way I'm thinking about it is anytime in any system we have a coordinate and want the entity + type there, we call this function. If it doesnt exist, we call terrain system and create it.

Do you think a simple rename of the function to something better like `getOrCreateGridEntity` is enough or are you thinking of something else?